vignettes/run-the-plan.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Run the drake plan"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Run the drake plan}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+1. [Write your own config file](https://kelshmo.github.io/sageseqr/articles/customize-config.html).
+
+2. Specify the active configuration by setting `R_CONFIG_ACTIVE`.
+
+```{r config-setup}
+Sys.setenv(R_CONFIG_ACTIVE = "default")
+```
+
+3. Load the `sageseqr` library and login to [Synapse](https://www.synapse.org/). `rnaseq_plan()` calls the arguments from the config file and creates the `drake` plan. Execute `drake::make(plan)` to compute.
+
+```{r run-plan}
+library(sageseqr)
+
+# Login to Synapse. Make a Synapse account and use synapser to login: https://r-docs.synapse.org/articles/manageSynapseCredentials.html
+synapser::synLogin()
+
+# Run the analysis
+plan <- sageseqr::rnaseq_plan(metadata_id = config::get("metadata")$synID,
```

Comment:
is it safe to assume that rnaseq_plan has been defined elsewhere? My knowledge of drake is super limited but the example I have noted generally use "drake_plan"

Reply:
It is since I wrote `rnaseq_plan` to wrap `drake_plan`!